services/web-app/data/libraries.js

```diff
@@ -392,9 +389,17 @@ const devLibraries = {
     org: 'carbon-design-system',
     repo: 'carbon',
     path: '/packages/styles',
-    sponsor: 'carbon'
-    // waiting for carbon react to be indexed
-    // group: carbonComponentsGroup
+    sponsor: 'carbon',
+    group: carbonComponentsGroup
+  },
+  'carbon-components': {
+    host: 'github.com',
+    org: 'carbon-design-system',
+    repo: 'carbon',
+    path: '/packages/components',
+    sponsor: 'carbon',
```

Comment:
do we still want to go with sponsor: carbon given our discussions today regarding carbon-components? CC: @mattrosno

Reply:
The Carbon core team does *not* sponsor the vanilla JS components. They have been community maintained for the last couple years.

Reply:
I think we can just remove the key then @andreancardona